content/millennium/r4/medications/medication-request.md

```diff
@@ -59,6 +59,20 @@ The following fields are returned if valued:
 
 <%= terminology_table(:medication_request_contained_medication, :r4) %>
 
+## Extensions
+
+* [NLLPrescriptionFormat](http://electronichealth.se/fhir/StructureDefinition/NLLPrescriptionFormat)
```

Comment:
I think these need to link out to the Swedish Specification. https://simplifier.net/guide/swedishnationalmedicationlist/MedicationRequest. If we look at examples of extensions on the Patient Resource the link on the extension name links to the specification.

Reply:
Fix for this: 
https://github.com/cerner/fhir.cerner.com/commit/a92347cb1bc1b50ba45b882417bda299dfc99c6b